index.html

```diff
@@ -18,6 +18,18 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200..1000;1,200..1000&display=swap"
     />
+
+    <!-- Umami tracking code -->
+
+    <!--
+    The integrity can not be checked when using cross origin
+    integrity="sha384-JyQQf9XxkMfVx/h+OQ+SxUZqLbL3p3KmWfnZ4LpQWzksmzHXESgXCk1/5AYxT9nR"
+    -->
```

Comment:
Remove?

Reply:
So, this integrity property was recommended by Sonarcloud, to ensure that the code that is downloaded actually meets what is expected, so it increases security because you can not download code that would have changed without you knowing it. 

The issue is that it does not work for cross-origin requests which is our case currently. It could work if we host umami ourselves AND we move to a single origin. So I can keep it in a comment, but yeah could also remove it.